niworkflows/interfaces/registration.py

```diff
@@ -347,8 +347,17 @@ def _post_run_hook(self, runtime):
 
 
 class EstimateReferenceImageInputSpec(BaseInterfaceInputSpec):
-    in_file = File(exists=True, mandatory=True, desc="4D EPI file")
-    sbref_file = File(exists=True, desc="Single band reference image")
+    in_file = InputMultiPath(File(exists=True),
```

Comment:
`InputMultiPath` is an old name that is being deprecated eventually. We should move to `InputMultiObject`.

```suggestion
    in_file = InputMultiObject(File(exists=True),
```